similarity/config/datastore.properties

```diff
@@ -0,0 +1,3 @@
+ENDPOINT	http://smaug.cs.illinois.edu:8080
+ACCESS-KEY	1I9SK88HMRAB9UWQ7URC
+SECRET-KEY	IsN/zQ99128acQyZF88ZdtRy+pO1eGEpubd6EhV0
```

Comment:
And drop this file from git history. These two lines are supposed to be secret. 

FYI @mssammon